tendermint/src/vote.rs

```diff
@@ -50,12 +50,76 @@ pub struct Vote {
 
     /// Signature
     pub signature: Option<Signature>,
+
+    /// Vote extension provided by the application.
+    /// Only valid for precommit messages.
+    pub extension: Vec<u8>,
+
+    /// Vote extension signature by the validator
+    /// Only valid for precommit messages.
+    pub extension_signature: Option<Signature>,
```

Comment:
Is it not worth stating that these fields are only relevant to CometBFT v0.38+?

Reply:
The whole type is new in 0.38, so I'll comment on that.

Reply:
https://github.com/informalsystems/tendermint-rs/pull/1312/commits/6a19f6c3b18ef9d7c24e89456eb6a4e2fe1a8879